base.css

```diff
@@ -296,6 +296,11 @@ div.result-message {
   padding: 0;
   margin: 1em 0;
 }
+.clear-search-btn {
+  width: 10%;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
You could style the selected card names and clear button inline (or not)

Reply:
Added better styling.